server/server/Routes/admin.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+
+const router = express.Router();
+const adminController = require("../controllers/adminCrud");
+
+router.post("/add", adminController.addNewItem);
+router.put("/update/:id", adminController.updateItem);
```

Comment:
since we can partially update our item, this should be a `PATCH` request
```suggestion
router.patch("/update/:id", adminController.updateItem);
```